package.json

```diff
@@ -82,5 +84,8 @@
     "deps/dicer/lib",
     "deps/streamsearch/",
     "deps/dicer/LICENSE"
-  ]
+  ],
+  "dependencies": {
```

Comment:
can you move this block closer to devDependencies?

Reply:
done